### DISCUSSION TITLE
Can not load radar image from radar.weather.gov/ridge/RadarImg/

### DISCUSSION THREAD

**Discussion by eepete** [weather-gov/api#237] [Q&A]
I have a system that has been able to load radar images for many years. Recently, that stoped working (no changes on code at my end). An example of what use to work is getting the radar Images from:
   https://radar.weather.gov/ridge/RadarImg/NCR/RAX_NCR_0.gif

Looking that the NWS site that explains how to get images at   https://www.weather.gov/jetstream/ridge_download
even the example in that document of:
     http://radar.weather.gov/ridge/RadarImg/N0R/FWS_N0R_0.gif
does not load. On  any browser (Chrome, Safari, Firefox) you see:
	Cannot GET /ridge/RadarImg/N0R/FWS_N0R_0.gif

Also tried with NCR, no luck. Any guidance on what has changed would be appreciated.

**Comment by jfbauer432** [comment]
They changed the radar site rather significantly as mentioned in #133 back in December 2020.  And it would appear no one updated that one page documenting the images.  You can see the available radar images at: https://radar.weather.gov/ridge/lite/

**Comment by eepete** [comment]
The images at  https://radar.weather.gov/ridge/lite/  are not the same as the "previous" images from .https://radar.weather.gov/ridge/RadarImg .  Those were images and an accompanying file (the .gfw) that you could download. That allowed the images to be overlaid on top of a map base. In my case, Google Maps with various map tile sets in addition to the ones Google provides.
  Are the "raw" images from the radar sites still available  or  is the new format all that can be provided ?

**Comment by eepete** [comment]
Enclosed are pictures of how the RAX_N0R_0.gif from the https://radar.weather.gov/ridge/lite/ looked like. Just the radar image, with the pixel that had no weather being transparent. Also enclosed is how this was used. First responders can get the radar images overlaid on the maps they are already using to see incident locations and responders/apparatus locations. This is why the new format at https://radar.weather.gov/ridge/RadarImg  isn't suitable as the only type of radar data, and it is why I'm asking how to get the "old" images that were previously available.

Original Images
![RAX_N0R_0](https://user-images.githubusercontent.com/78659904/107303644-7ac9be80-6a4d-11eb-8a52-1c57a14e6508.gif)

Example of overlaying radar on first responders mapping
![radar2](https://user-images.githubusercontent.com/78659904/107303700-9208ac00-6a4d-11eb-8b94-8e173f97ffe6.png)

**Comment by scadergit** [comment]
Fyi, this is the API discussion board, which is not related to the GIS services for the NWS, but happy to help since you're here. If you need additional assistance, please refer to [their web site for more information](https://www.weather.gov/gis/WebServices). 

As @jfbauer432 noted, the old images are no longer available. In the previous version, the "lite" images were intended to provide a means for 3rd party developers to integrate radar into their applications (now they are simply a means to quickly insert static images). That solution has been replaced with a geoserver that is far more feature rich for that purpose. You can technically pull the same static images directly out of the geoserver, but I would also encourage you to explore how the geoserver can integrate with Google maps as it will allow a more dynamic application.

**Comment by eepete** [comment]
Thank you for taking to time to point me at the correct resources.

  FWIW, I'm trying to move away from Google Maps. While the product is good, it requires the ability to have a high speed connection to the Internet which is not possible in many types of disaster response. It is a common problem that many tools for responders are unusable when there is no high speed connectivity, which is a real problem. I've always used a minimum of the Google features, and am slowly working towards a solution based on the Open Street Layers software. If Google would just allow local storage of the Javascript library file, life would be good. But no.

  There does not appear to be a guide for migrating from the "old" to the "new GIS" service, I'll contact them. In a situation similar to the above, much of the GIS world is driven by ESRI GIS software ecosystem, one that is neither compact, frugal with bandwidth or able to function in isolated situations. The rise of dependance on certificate based RESTful transactions is another challenge.

  Modern Problems, but if you're operating in an area with massive infrastructure failure or no infrastructure (woodland fires), a relevant one.

  Thank you again for your reply.

 - pete

Pete Hallenbeck
Efland, NC
pete@eflandfd.org
